components/browser/state/src/main/java/mozilla/components/browser/state/state/SessionState.kt

```diff
@@ -24,6 +25,7 @@ interface SessionState {
     val engineState: EngineState
     val extensionState: Map<String, WebExtensionState>
     val contextId: String?
+    var lastAccess: Long
```

Comment:
It would be better if we you `val` as every time we change this object we are mutating it.

Reply:
+1